service/worker/archiver/activities.go

```diff
@@ -82,15 +82,6 @@ func uploadHistoryActivity(ctx context.Context, request ArchiveRequest) (err err
 		NextEventID:          request.NextEventID,
 		CloseFailoverVersion: request.CloseFailoverVersion,
 	}, carchiver.GetHeartbeatArchiveOption(), carchiver.GetNonRetriableErrorOption(errUploadNonRetriable))
-	if err == nil {
```

Comment:
I think we want this code?

Reply:
Added them back. Idea is it's ok to log in both the activity and in archiver.